test/mavsdk_tests/autopilot_tester.h

```diff
@@ -128,24 +128,27 @@ class AutopilotTester
 			uint32_t start_time = _info->get_flight_information().second.time_boot_ms;
 
 			while (!fun()) {
-				std::this_thread::sleep_for(duration_ms / 1000);
+				std::this_thread::sleep_for(duration_ms / 100);
 
```

Comment:
This is wrong now.

Reply:
It is not - the sleep quantum is completely independent (the way it is implemented now).